### PR TITLE
L2cap fixes

### PR DESCRIPTION
**Pull request by KKopyscinski**

### CODE REVIEW

File: pybtp/btp.py [hunk 1]
```diff
@@ -3269,8 +3269,8 @@ def l2cap_send_data(chan_id, val, val_mtp=None):
     stack.l2cap.tx(chan_id, val)
 
 
-def l2cap_listen(psm, transport, mtu=0, response=0):
-    logging.debug("%s %r %r", l2cap_le_listen.__name__, psm, transport)
+def l2cap_listen(psm, transport, mtu=0, req_sec=L2capSecLevels.no_sec, req_key_size=0, response=0,):
```
Comment (michal-narajowski) [inline pybtp/btp.py:3272]: `,` at the end

File: pybtp/types.py [hunk 2]
```diff
@@ -241,3 +241,8 @@ class Perm:
     @staticmethod
     def decode(perm):
         return decode_flag_name(perm, Perm.names)
+
+class L2capSecLevels:
+    no_sec = 0x01
+    authen = 0x02
+    author = 0x03
```
Comment (michal-narajowski) [inline pybtp/types.py:248]: New line

File: ptsprojects/zephyr/l2cap.py [hunk 3]
```diff
@@ -294,14 +294,25 @@ def test_cases(pts):
                   pre_conditions_eatt,
                   generic_wid_hdl=l2cap_wid_hdl),
         ZTestCase("L2CAP", "L2CAP/ECFC/BV-11-C",
-                  # this test requires EATT enabled, don't create L2CAP server
-                  common,
+                  common +
+                  [TestFunc(lambda: pts.update_pixit_param("L2CAP", "TSPX_iut_supported_max_channels", "2")),
```
Comment (michal-narajowski) [inline ptsprojects/zephyr/l2cap.py:298]: I would extract common code and create a new list similar to `pre_conditions_eatt`.

**Review by michal-narajowski — CHANGES_REQUESTED** [review]
Minor comments, otherwise LGTM

File: ptsprojects/zephyr/l2cap.py (hunk 3 above)
Comment (KKopyscinski) [reply ptsprojects/zephyr/l2cap.py]: Done :)

**Review by michal-narajowski — APPROVED** [review]

**Pull request merged**